packages/wonder-blocks-dropdown/components/action-menu.test.js

```diff
@@ -7,13 +7,10 @@ import ActionItem from "./action-item.js";
 import OptionItem from "./option-item.js";
 import SeparatorItem from "./separator-item.js";
 import ActionMenu from "./action-menu.js";
-
-const keyCodes = {
-    enter: 13,
-    space: 32,
-};
+import {keyCodes} from "../util/constants.js";
 
 describe("ActionMenu", () => {
+    global.scrollTo = jest.fn();
```

Comment:
Hmm... this doesn't feel right. I think this needs to be in a `beforeEach` and then removed again in the `afterEach` like we do now for mount. Otherwise this will blow away the global which might have negative ramifications somehow? (Even though we're just running this in Node)

Alternatively, you could change the code itself to do something like:

```
if (window.scrollTo) {
    window.scrollTo(...);
}
```

That way we don't have to stub it out at all.

Reply:
Using `window.scrollTo` is better, changed!